app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -148,10 +149,10 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   }
 
   private fun createRecyclerViewAdapter(): BindableAdapter<ProfileChooserUiModel> {
-    return BindableAdapter.MultiTypeBuilder
-      .newBuilder<ProfileChooserUiModel, ProfileChooserUiModel.ModelTypeCase>(
-        ProfileChooserUiModel::getModelTypeCase
-      )
+    return multiTypeAdapterBuilder.create<ProfileChooserUiModel,
```

Comment:
Ditto for this variable name--can we line wrap this more cleanly?

Reply:
Patched.